src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -1,6 +1,7 @@
 # cloud.yaml contains the default configuration that's made available on the
 # internet.
 client:
+  firetweetversion: "0.0.6"
```

Comment:
It might be too tightly coupled, as we need to rerun `genconfig` for each new FT version. Alternative would be to put a `version.txt` besides `firetweet.apk`, and check the content directly. It's rather easy to update the txt file at the same time when updating apk.